src/core/utils.ts

```diff
@@ -173,18 +173,18 @@ export function base64UrlEncode(buffer: ArrayBuffer): string {
     return urlVariant;
 }
 
-export function registerServiceWorker(
-    path: string
-): Promise<ServiceWorkerRegistration> {
+export async function registerServiceWorker(workerPath: string){
```

Comment:
@k-antipochkin can you check tooling for Prettier? looks like formatting hasn't been applied on save in these files. Thanks!

Reply:
Formatted the file, those lines remained the same though

Reply:
Cool, think some spacings tweaked which is good, yeah it doesn't always line break in prettier unless over some soft wrap I think.